HUD/UseFakeMpCash.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: HUD
+aliases: ["0x170F541E1CADD1DE"]
+---
+## USE_FAKE_MP_CASH
+
+```c
+// 0x170F541E1CADD1DE 0x6253B9D7
+void USE_FAKE_MP_CASH(BOOL p0);
+```
+
+```
```

Comment:
Would be nice if these descriptions were also changed to normal descriptions instead of code blocks, but I'll allow it for now.